src/themes/base/siteVariables.ts

```diff
@@ -8,11 +8,13 @@ export { colors, contextualColors, emphasisColors, naturalColors, colorScheme }
 // FONT SIZES
 //
 export const fontSizes = {
+  smallest: '0.8rem',
   smaller: '1rem',
   small: '1.2rem',
   medium: '1.4rem',
   large: '1.8rem',
```

Comment:
frankly, to me it is not absolutely intuitive whether `large` or `larger` is bigger while just having a markup:

```jsx
<Button size='larger' />
```

### Further thoughts

It would be much better to have some range where each value's name would immediately deliver insights on what is the name of next larger item and next smaller one, like, say, in the following case:

```js
xx-small    // or x2-small, to reduce amount of x's
x-small
medium
large
x-large
xx-large
```

Amongst other things it would be great if this range would be extendable, so that client's theme may easily provide additional sizes to it while maintaining the overall order of values.

With this order being established, we might be able to decently handle scenarios when one theme, with wider range of sizes, is switched on another one where this range is smaller:

```jsx
// original theme
const fontSizes = {
  xx-small
  x-small
  medium
  large
  x-large
  xx-large
}

// theme to switch on - note, no values for xx-large/small
const fontSizes = {
  x-small
  medium
  large
  x-large
}
```

The idea here is just to allow next theme to choose nearest size to the one that is requested (so, for `xx-large` new theme will use its `x-large`) - and this is pretty much the same what is happening with fonts in Web (both speaking of sizes and weights).

### To conclude

This was just one of the approaches we might think about - definitely, there may be lots of other ones. At the same time, if we would like to provide client with an intuitive range of values, as well as to support theme switching between two that have different set of sizes defined, we would need to satisfy the following conditions:
- names of values suggest their order
- it is possible to infer next closest value from the name of each one
- there is an extendable range of values

Reply:
> The sizing scheme we decided to go with for now is: smallest, smaller, small, medium, large, larger, largest.

#640

> For themes that may want 5 sizes, just have the largest/smallest sizes use the same values as larger/smaller to limit them.

https://github.com/stardust-ui/react/issues/136#issuecomment-447152727

Reply:
thanks for these references and clarifications!